lang/python/python3-readline/Makefile

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (C) 2015 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python3-readline
+PKG_VERSION:=6.2.4.1
+PKG_RELEASE:=$(AUTORELEASE)
+
+PYPI_NAME:=readline
```

Comment:
Why are you trying to add deprecated package? From [readline's pypi.org](https://pypi.org/project/readline/):
**WARNING: THIS PACKAGE IS DEPRECATED! It has been renamed to GNUREADLINE to resolve a name clash with the standard library module. The new PyPI page is here.**

While looking at [gnureadline's pypi.org](https://pypi.org/project/gnureadline/), why do we need it?